kinbot/kb.py

```diff
@@ -176,18 +185,47 @@ def main():
         logging.info('Starting the search for homolytic scission products')
         well0.homolytic_scissions = HomolyticScissions(well0, par, qc)
         well0.homolytic_scissions.find_homolytic_scissions()
-  
     # initialize the master equation instance
-    mess = MESS(par, well0)
-    mess.write_input()
-    #mesmer = MESMER(par, well0)
-    #mesmer.write_input()
+
+    # check to see if uq analysis is turned on
+    if par.par['me'] == 1:
+        logging.info('ME turned on')
+        if uq == 0:
+            n=1
+            logging.info('uq turned off')
+            mess = MESS(par, well0)
+            mess.write_input(uq,n)
+        elif uq == 1:
+            logging.info('uq turned on')
+            n=uq_n
+            mess = MESS(par, well0)
+            mess.write_input(uq,n)
+        else:
+            logging.error('Cannot recognize uq code {}'.format(par.par['uq']))
+    else:
+        logging.info('ME turned off')
+    
+    #TO DO: CODE UQ into MESMER
+
+    mesmer = MESMER(par, well0)
+    mesmer.write_input()
+    #mess = MESS(par, well0)
+    #mess.write_input()
```

Comment:
can be deleted

Reply:
I deleted a few lines here related to UQ with Mesmer that were commented out as well.